File: src/main/java/com/heekng/celloct/CelloctApplication.java
package com.heekng.celloct;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
public class CelloctApplication {

	public static void main(String[] args) {
		SpringApplication.run(CelloctApplication.class, args);
	}

}

File: src/main/kotlin/com/heekng/celloct/CelloctApplication.kt
package com.heekng.celloct

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
class CelloctApplication

fun main(args: Array<String>) {
    runApplication<CelloctApplication>(*args)
}